src/createJestConfig.ts

```diff
@@ -1,15 +1,30 @@
 import { Config } from '@jest/types';
+import { createRequire } from 'module';
```

Comment:
Per [the docs](https://nodejs.org/api/module.html#module_module_createrequire_filename) you linked to in https://github.com/formium/tsdx/issues/688#issuecomment-724892954, `createRequire` only works Node 12+. TSDX supports Node 10+, so this API cannot be used.

Reply:
Switched to `create-require` which works on node 10.